<Type Name="StackOverflowException" FullName="System.StackOverflowException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="82d7baf0a115d57b8e720be59cdabe15ed700b9b" /><Meta Name="ms.sourcegitcommit" Value="b6b9d5f3f9c7eb43f14a8a1a68d044603cb8ed73" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="ja-JP" /><Meta Name="ms.lasthandoff" Value="05/18/2020" /><Meta Name="ms.locfileid" Value="83520766" /></Metadata><TypeSignature Language="C#" Value="public sealed class StackOverflowException : SystemException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi sealed beforefieldinit StackOverflowException extends System.SystemException" FrameworkAlternate="net-5.0;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netstandard-2.0;netstandard-2.1" />
  <TypeSignature Language="DocId" Value="T:System.StackOverflowException" />
  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class StackOverflowException&#xA;Inherits SystemException" />
  <TypeSignature Language="C++ CLI" Value="public ref class StackOverflowException sealed : SystemException" />
  <TypeSignature Language="F#" Value="type StackOverflowException = class&#xA;    inherit SystemException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable sealed beforefieldinit StackOverflowException extends System.SystemException" FrameworkAlternate="netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>1.0.5000.0</AssemblyVersion>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>netstandard</AssemblyName>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
    <AssemblyVersion>2.1.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>System.Runtime</AssemblyName>
    <AssemblyVersion>4.2.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.1.0</AssemblyVersion>
    <AssemblyVersion>4.2.2.0</AssemblyVersion>
    <AssemblyVersion>5.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <TypeForwardingChain>
    <TypeForwarding From="netstandard" FromVersion="2.1.0.0" To="System.Runtime" ToVersion="5.0.0.0" FrameworkAlternate="net-5.0" />
  </TypeForwardingChain>
  <Base>
    <BaseTypeName>System.SystemException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0;netframework-4.8">
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
    <Attribute FrameworkAlternate="netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0">
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="1c2ba-101">入れ子になったメソッド呼び出しが多くなりすぎ、実行スタックがオーバーフローした場合にスローされる例外。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-101">The exception that is thrown when the execution stack overflows because it contains too many nested method calls.</span></span> <span data-ttu-id="1c2ba-102">このクラスは継承できません。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-102">This class cannot be inherited.</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="1c2ba-103">`StackOverflowException` は、実行スタックオーバーフローエラーに対してスローされます。通常、非常に深い再帰または無制限の再帰の場合に発生します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-103">`StackOverflowException` is thrown for execution stack overflow errors, typically in case of a very deep or unbounded recursion.</span></span> <span data-ttu-id="1c2ba-104">したがって、コードに無限ループや無限再帰がないことを確認してください。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-104">So make sure your code doesn't have an infinite loop or infinite recursion.</span></span>
  
 <span data-ttu-id="1c2ba-105">`StackOverflowException` では、値0x800703E9 を持つ HRESULT COR_E_STACKOVERFLOW を使用します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-105">`StackOverflowException` uses the HRESULT COR_E_STACKOVERFLOW, which has the value 0x800703E9.</span></span> <span data-ttu-id="1c2ba-106"><xref:System.Reflection.Emit.OpCodes.Localloc> 中間言語 (IL) 命令が `StackOverflowException`をスローします。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-106">The <xref:System.Reflection.Emit.OpCodes.Localloc> intermediate language (IL) instruction throws `StackOverflowException`.</span></span> <span data-ttu-id="1c2ba-107">`StackOverflowException` オブジェクトの初期プロパティ値の一覧については、「<xref:System.StackOverflowException.%23ctor%2A> コンストラクター」を参照してください。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-107">For a list of initial property values for a `StackOverflowException` object, see the <xref:System.StackOverflowException.%23ctor%2A> constructors.</span></span>  
  
 <span data-ttu-id="1c2ba-108">.NET Framework 2.0 以降では、`try`/`catch` ブロックを使用して `StackOverflowException` オブジェクトをキャッチすることはできません。また、対応するプロセスは既定で終了します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-108">Starting with the .NET Framework 2.0, you can't catch a `StackOverflowException` object with a `try`/`catch` block, and the corresponding process is terminated by default.</span></span> <span data-ttu-id="1c2ba-109">そのため、スタックオーバーフローを検出して回避するコードを記述する必要があります。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-109">Consequently, you should write your code to detect and prevent a stack overflow.</span></span> <span data-ttu-id="1c2ba-110">たとえば、アプリが再帰に依存している場合は、カウンターまたは状態条件を使用して再帰ループを終了します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-110">For example, if your app depends on recursion, use a counter or a state condition to terminate the recursive loop.</span></span>
<span data-ttu-id="1c2ba-111">この手法の図解については、「[例](#examples)」を参照してください。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-111">See the [Examples](#examples) section for an illustration of this technique.</span></span>
 
> [!NOTE]
>  <span data-ttu-id="1c2ba-112">`StackOverflowException` をスローするメソッドに <xref:System.Runtime.ExceptionServices.HandleProcessCorruptedStateExceptionsAttribute> 属性を適用しても効果はありません。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-112">Applying the <xref:System.Runtime.ExceptionServices.HandleProcessCorruptedStateExceptionsAttribute> attribute to a method that throws a `StackOverflowException` has no effect.</span></span> <span data-ttu-id="1c2ba-113">ただし、ユーザーコードから例外を処理することはできません。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-113">You still cannot handle the exception from user code.</span></span>

 <span data-ttu-id="1c2ba-114">アプリが共通言語ランタイム (CLR) をホストしている場合は、CLR がスタックオーバーフロー例外が発生したアプリケーションドメインをアンロードし、対応するプロセスを続行するように指定できます。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-114">If your app hosts the common language runtime (CLR), it can specify that the CLR should unload the application domain where the stack overflow exception occurs and let the corresponding process continue.</span></span> <span data-ttu-id="1c2ba-115">詳細については、「 [ICLRPolicyManager インターフェイス](~/docs/framework/unmanaged-api/hosting/iclrpolicymanager-interface.md)」を参照してください。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-115">For more information, see [ICLRPolicyManager Interface](~/docs/framework/unmanaged-api/hosting/iclrpolicymanager-interface.md).</span></span>

## Examples

 <span data-ttu-id="1c2ba-116">次の例では、カウンターを使用して、`Execute` メソッドへの再帰呼び出しの回数が、MAX_RECURSIVE_CALLS 定数によって定義された最大値を超えないようにしています。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-116">The following example uses a counter to ensure that the number of recursive calls to the `Execute` method do not exceed a maximum defined by the MAX_RECURSIVE_CALLS constant.</span></span>
  
 [!code-csharp[System.StackOverflowException.Class#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.stackoverflowexception.class/cs/example1a.cs#1)]
 [!code-vb[System.StackOverflowException.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.stackoverflowexception.class/vb/example1a.vb#1)]  

 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
    <related type="Article" href="~/docs/standard/exceptions/index.md"><span data-ttu-id="1c2ba-117">例外の処理とスロー</span><span class="sxs-lookup"><span data-stu-id="1c2ba-117">Handling and Throwing Exceptions</span></span></related>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="1c2ba-118"><see cref="T:System.StackOverflowException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-118">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public StackOverflowException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.StackOverflowException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; StackOverflowException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
        <AssemblyVersion>4.2.2.0</AssemblyVersion>
        <AssemblyVersion>5.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="1c2ba-119"><see cref="T:System.StackOverflowException" /> クラスの新しいインスタンスの <see cref="P:System.Exception.Message" /> プロパティに、"要求された操作によってスタック オーバーフローが発生しました。" などのエラーを説明するシステムから提供されるメッセージを設定して、その新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-119">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class, setting the <see cref="P:System.Exception.Message" /> property of the new instance to a system-supplied message that describes the error, such as "The requested operation caused a stack overflow."</span></span> <span data-ttu-id="1c2ba-120">このメッセージには、システムの現在のカルチャが考慮されます。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-120">This message takes into account the current system culture.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="1c2ba-121">次の表に、<xref:System.StackOverflowException> のインスタンスに対するプロパティの初期値を示します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-121">The following table shows the initial property values for an instance of <xref:System.StackOverflowException>.</span></span>  
  
|<span data-ttu-id="1c2ba-122">プロパティ</span><span class="sxs-lookup"><span data-stu-id="1c2ba-122">Property</span></span>|<span data-ttu-id="1c2ba-123">値</span><span class="sxs-lookup"><span data-stu-id="1c2ba-123">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="1c2ba-124">[https://login.microsoftonline.com/consumers/](`null`)</span><span class="sxs-lookup"><span data-stu-id="1c2ba-124">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="1c2ba-125">ローカライズされたエラー メッセージの文字列です。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-125">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public StackOverflowException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.StackOverflowException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; StackOverflowException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new StackOverflowException : string -&gt; StackOverflowException" Usage="new System.StackOverflowException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
        <AssemblyVersion>4.2.2.0</AssemblyVersion>
        <AssemblyVersion>5.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="1c2ba-126">エラーを説明する <see cref="T:System.String" />。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-126">A <see cref="T:System.String" /> that describes the error.</span></span> <span data-ttu-id="1c2ba-127">メッセージの内容は、ユーザーが理解できる内容にします。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-127">The content of message is intended to be understood by humans.</span></span> <span data-ttu-id="1c2ba-128">このコンストラクターの呼び出し元は、この文字列が現在のシステム環境向けにローカライズされていることを確認する必要があります。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-128">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span></param>
        <summary><span data-ttu-id="1c2ba-129">指定したエラー メッセージを使用して、<see cref="T:System.StackOverflowException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-129">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class with a specified error message.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="1c2ba-130">次の表に、<xref:System.StackOverflowException> のインスタンスに対するプロパティの初期値を示します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-130">The following table shows the initial property values for an instance of <xref:System.StackOverflowException>.</span></span>  
  
|<span data-ttu-id="1c2ba-131">プロパティ</span><span class="sxs-lookup"><span data-stu-id="1c2ba-131">Property</span></span>|<span data-ttu-id="1c2ba-132">値</span><span class="sxs-lookup"><span data-stu-id="1c2ba-132">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="1c2ba-133">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-133">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="1c2ba-134">エラー メッセージ文字列です。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-134">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public StackOverflowException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.StackOverflowException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; StackOverflowException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new StackOverflowException : string * Exception -&gt; StackOverflowException" Usage="new System.StackOverflowException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
        <AssemblyVersion>4.2.2.0</AssemblyVersion>
        <AssemblyVersion>5.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="1c2ba-135">例外の原因を示すエラー メッセージです。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-135">The error message that explains the reason for the exception.</span></span></param>
        <param name="innerException"><span data-ttu-id="1c2ba-136">現在の例外の原因になった例外です。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-136">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="1c2ba-137"><paramref name="innerException" /> パラメーターが null 参照 (Visual Basic の場合は <see langword="Nothing" />) でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-137">If the <paramref name="innerException" /> parameter is not a null reference (<see langword="Nothing" /> in Visual Basic), the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="1c2ba-138">指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を使用して、<see cref="T:System.StackOverflowException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-138">Initializes a new instance of the <see cref="T:System.StackOverflowException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="1c2ba-139">前の例外の直接の結果としてスローされる例外は、<xref:System.Exception.InnerException%2A> プロパティに前の例外への参照を含んでいる場合があります。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-139">An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="1c2ba-140"><xref:System.Exception.InnerException%2A> プロパティは、コンストラクターに渡されたものと同じ値を返します。内部例外値がコンストラクターに渡されなかった場合、<xref:System.Exception.InnerException%2A> プロパティはnull 参照 (Visual Basic の場合は `Nothing`) を返します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-140">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a null reference (`Nothing` in Visual Basic) if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="1c2ba-141">次の表に、<xref:System.StackOverflowException> のインスタンスに対するプロパティの初期値を示します。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-141">The following table shows the initial property values for an instance of <xref:System.StackOverflowException>.</span></span>  
  
|<span data-ttu-id="1c2ba-142">プロパティ</span><span class="sxs-lookup"><span data-stu-id="1c2ba-142">Property</span></span>|<span data-ttu-id="1c2ba-143">値</span><span class="sxs-lookup"><span data-stu-id="1c2ba-143">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="1c2ba-144">内部例外の参照です。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-144">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="1c2ba-145">エラー メッセージ文字列です。</span><span class="sxs-lookup"><span data-stu-id="1c2ba-145">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
        <related type="Article" href="~/docs/standard/exceptions/index.md"><span data-ttu-id="1c2ba-146">例外の処理とスロー</span><span class="sxs-lookup"><span data-stu-id="1c2ba-146">Handling and Throwing Exceptions</span></span></related>
      </Docs>
    </Member>
  </Members>
</Type>
